code/upaya/Procfile

```diff
@@ -0,0 +1,5 @@
+web: bundle exec rake db:migrate
+web: bundle exec rake assets:precompile
+web: bundle exec puma -C config/puma.rb
+#INLINE method to startup puma web: bundle exec puma -t 5:5 -p ${PORT:-3000} -e ${RACK_ENV:-development}
+resque: env TERM_CHILD=1 bundle exec rake resque:work
```

Comment:
It might be worth looking at how e-manifest does this, especially:

https://github.com/18F/e-manifest/blob/master/e-manifest.manifest.yml
https://github.com/18F/e-manifest/blob/master/script/worker-start
https://github.com/18F/e-manifest/blob/master/script/web-start

since presumably we'll want different web and worker instances. Also, this is handy:

https://github.com/18F/e-manifest/blob/master/lib/tasks/cf.rake

since when running multiple instances, you only want the start-up rake tasks executing on one of them.